pkg/auth/handler/webapp/authflow_controller.go

```diff
@@ -925,7 +926,7 @@ func (c *AuthflowController) makeErrorResult(w http.ResponseWriter, r *http.Requ
 
 	switch {
 	case errors.Is(err, authflow.ErrFlowNotFound):
-		u.Path = "/errors/error"
+		c.Navigator.NavigateNonRecoverableError(&u)
 		return nonRecoverable()
 	case user.IsAccountStatusError(err):
```

Comment:
The navigator should be reponsible to navigate to `webapp.AuthflowRouteNoAuthenticator` and `webapp.AuthflowRouteAccountStatus` too.